ci/build.sh

```diff
@@ -143,7 +143,7 @@ if [ -n "$DOCKER_PASSWORD" ] && [ -n "$TRAVIS_TAG" ] && [[ $TRAVIS_TAG != *alpha
   # Push new helm release
   git checkout -- .
   sudo helm init --client-only
-  sudo helm package deploy/helm/sumologic --version=$VERSION
+  sudo helm package deploy/helm/sumologic --version=$VERSION --app-version=$VERSION
```

Comment:
why do we need to change this?

Reply:
Helm seems to be doing this already, but it makes me nervous relying on that undocumented behavior that could change.  So to be explicit I added this.